test/cpp/end2end/thread_stress_test.cc

```diff
@@ -339,8 +341,9 @@ class AsyncClientEnd2endTest : public ::testing::Test {
   void TearDown() override {
     void* ignored_tag;
     bool ignored_ok;
-    while (cq_.Next(&ignored_tag, &ignored_ok))
+    while (cq_.Next(&ignored_tag, &ignored_ok)) {
       ;
```

Comment:
No need for this anymore, now that there are braces around it.

Reply:
Done